ui/app/selectors/selectors.js

```diff
@@ -203,11 +203,11 @@ export function accountsWithSendEtherInfoSelector(state) {
   const accounts = getMetaMaskAccounts(state)
   const identities = getMetaMaskIdentities(state)
 
-  const accountsWithSendEtherInfo = Object.entries(identities).map(
-    ([key, identity]) => {
+  const accountsWithSendEtherInfo = Object.entries(identities)
+    .map(([key, identity]) => {
       return { ...identity, ...accounts[key] }
-    },
-  )
+    })
+    .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
This selector is used in a number of places. I wonder if this sort breaks any important defaults of the sort in those places? While it depends on the browser, I think in most cases (on prod) the order of these accounts would default to the order in which the accounts were added. That is, the order in which they are generated from the private key.

This will change that ordering and will change user experience across a few different components.

The safest thing to do here is just to sort the instance relevant to the "My accounts list". Whether we should change ordering elsewhere can be considered outside this issue.

Reply:
Fixed in: e170509